src/test/java/com/epam/repository/HotelDAOTest.java

```diff
@@ -44,7 +44,7 @@ public void buildHotel(){
         when(resultSet.getString("name")).thenReturn(testHotelName);
         when(resultSet.getString("city")).thenReturn(testHotelCity);
         when(resultSet.getString("country")).thenReturn(testHotelCountry);
-        when(resultSet.getInt("numberOfStars")).thenReturn(testHotelStars);
+        when(resultSet.getInt("number_Of_Stars")).thenReturn(testHotelStars);
```

Comment:
It seems like Of & Starts with the first letter in upper case, is it okay?

Reply:
done